pylint/checkers/utils.py

```diff
@@ -233,6 +233,26 @@ def is_inside_lambda(node: astroid.node_classes.NodeNG) -> bool:
     return False
 
 
+def is_call_of_name(node: astroid.node_classes.NodeNG, name: str) -> bool:
```

Comment:
These are currently consumed in the refactoring checker, let's keep it there for now, as otherwise they would be considered public utilities.